recipes/ozzy-pic/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "ozzy-pic" %}
+{% set version = "1.0.5" %}
+{% set python_min = python_min|default("3.10") %}
```

Let's try just this:

```suggestion
{% set python_min = "3.10" %}
```